azure-documentdb-spring-boot-starter/README.md

```diff
@@ -97,6 +97,11 @@ public class SampleApplication implements CommandLineRunner {
 ```
 Autowired UserRepository interface, then can do save, delete and find operations.
 
+### BI data collection
+Microsoft would like to collect data about how users use this Spring boot starter. Data collected is like hash of mac address. Microsoft uses this information to improve our tooling experience. Participation is voluntary. If you don't want to participate, just simple disable it by setting below configuration in `application.properties`.
```

Comment:
Data collected is like hash of mac address.
->
The only collected data is the hashed mac address. 

simple -> simply